### DISCUSSION TITLE
Project status

### DISCUSSION THREAD

**Discussion by Powell-v2** [nativewind/nativewind#505] [General]
Hi @marklawlor 👋 

I was wondering what is the current status of the project? I would like to start using the library in one of my apps, but date of the latest release (September 2022), along with the number of unanswered issues / discussions / PRs, makes me think that the project is becoming stale.

Thanks you upfront for clarifying!

P.S. I might be interested in becoming a maintainer if you are looking for one. 

Best,
Pavel Y.

**Comment by lgoudriaan** [comment]
See https://github.com/marklawlor/nativewind/issues/464

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
